app/src/main/java/com/orange/ods/app/ui/MainScreen.kt

```diff
@@ -195,30 +195,30 @@ private fun AppBarTabs(appBarTabsState: AppBarTabsState) {
             // Do not use tabs directly because this is a SnapshotStateList
             // Thus its value can be modified and can lead to crashes if it becomes empty
             val tabs = tabs.toList()
-            val tabIconPosition =
-                if (tabIconType.value == MainTabsCustomizationState.TabIconType.Leading && tabTextEnabled.value) OdsTabRow.Tab.Icon.Position.Leading else OdsTabRow.Tab.Icon.Position.Top
+            val tabsIconPosition =
+                if (tabsIconPosition.value == MainTabsCustomizationState.TabsIconPosition.Leading && tabTextEnabled.value) OdsTabRow.Tab.Icon.Position.Leading else OdsTabRow.Tab.Icon.Position.Top
```

Comment:
I'm not 100% sure but checking `tabTextEnabled.value` might not be mandatory here. If I'm correct this case is managed by the tab component itself.